### DISCUSSION TITLE
Has Avian an Event Reader for Collisions?

### DISCUSSION THREAD

**Discussion by jpx40** [avianphysics/avian#587] [Q&A] [answered]
Has Avian an Event Reader for Collisions?

**Comment by Jondolf** [comment, marked as answer]
Yes, see [docs](https://docs.rs/avian3d/0.1.2/avian3d/collision/contact_reporting/struct.ContactReportingPlugin.html#collision-events)

**Reply by jpx40** [reply]
thanks

**Reply by jpx40** [reply]
why does have a collision just 2 fields for Entities
